RedHatInsights/compliance-frontend
# feat(RHINENG-10391): migrate Reports page to REST

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist
This migrates the Reports page to use REST API with a flag. It also includes serialiser implementation using lodash set function.

To test:
1. Visit reports page with compliance-api-v2 flag disabled
2. Observe that there are no changes in the behavior and grapgQL is used to fetch data
3. Enable the flag
4. Observe that now REST api is used 
5. Observe that table renders all data
6. Observe that filtering, pagination and other functionalities yet disabled

## Secure Coding Checklist
